src/webots/vrml/WbProtoModel.cpp

```diff
@@ -170,9 +170,8 @@ WbProtoModel::WbProtoModel(WbTokenizer *tokenizer, const QString &worldPath, con
           insideTemplateStatement = false;
         else if (c == '"' && pc != '\\')
           insideDoubleQuotes = !insideDoubleQuotes;
-        else if (!insideTemplateStatement && c == '#' && !insideDoubleQuotes && mTemplateLanguage == "lua")
```

Comment:
We should get rid of this `mTemplateLanguage` member field since its value will always be `"javascript"` now.

Reply:
No, we have no such plans.